apps/web/src/pages/api/profile/[user]/tokens.tsx

```diff
@@ -0,0 +1,38 @@
+import axios from 'axios'
+import { NextApiRequest, NextApiResponse } from 'next'
+
+import { MyDaosResponse } from 'src/data/graphql/requests/daoQuery'
+import { TokensQueryResponse, tokensQuery } from 'src/data/graphql/requests/tokensQuery'
+import { getBaseUrl } from 'src/utils/baseUrl'
+
+export interface UserTokensResponse {
```

Comment:
it looks like this file is a tsx file even thought it's an api route. is that intended? I also noticed `api/profile/[user]/daos` is the same.